core/src/main/scala/cats/AutoVariance.scala

```diff
@@ -0,0 +1,10 @@
+package cats
+
+trait AutoVariance {
+  implicit def autoWidenFunctor[F[_]: Functor, A, B >: A](fa: F[A]): F[B] = Functor[F].widen(fa)
+
+  implicit def autoNarrowContravariant[F[_]: Contravariant, A, B <: A](fa: F[A]): F[B] = Contravariant[F].narrow(fa)
+
+  implicit def autoLeftWidenBifunctor[F[_, _]: Bifunctor, A, B >: A, C](fac: F[A, C]): F[B, C] =
+    Bifunctor[F].leftWiden(fac)
```

Comment:
I think this needs to be adjusted to a lower priority variant (so it doesn't conflict with `autoWidenFunctor`) which widens *both* parameters simultaneously, since the compiler will not chain coercions.

Reply:
ah, in the case where you'd like to auto-widen both parameters? Good call!